### PR TITLE
Fixed : [CI] DownsampleActionSingleNodeTests » testCannotRollupWhileOtherRollupInProgress

### PR DESCRIPTION
**Pull request by weizijun**
The reason why this test some times failed is that, the next rollup call is so close to the first rollup call, it may case the first rollup failed.
I add a check, that the first rollup has already created the rollup index, then call the next rollup.

### CODE REVIEW

**Comment by csoulios** [comment]
@elasticmachine ok to test

**Comment by weizijun** [comment]
@elasticmachine update branch

**Comment by csoulios** [comment]
@elasticmachine run elasticsearch-ci/part-1

**Comment by csoulios** [comment]
@elasticmachine run elasticsearch-ci/part-1

**Comment by csoulios** [comment]
Fixes https://github.com/elastic/elasticsearch/issues/90197

**Comment by weizijun** [comment]
@elasticmachine run elasticsearch-ci/part-1

**Comment by weizijun** [comment]
@elasticmachine update branch

**Comment by csoulios** [comment]
@elasticmachine update branch

File: x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/downsample/DownsampleActionSingleNodeTests.java [hunk 1]
```diff
@@ -525,6 +524,13 @@ public void onFailure(Exception e) {
             }
         };
         client().execute(DownsampleAction.INSTANCE, new DownsampleAction.Request(sourceIndex, rollupIndex, config), rollupListener);
+        assertBusy(() -> {
+            try {
+                assertEquals(client().admin().indices().prepareGetIndex().addIndices(rollupIndex).get().getIndices().length, 1);
+            } catch (IndexNotFoundException e) {
+                fail("rollup index has not created");
```
Comment (csoulios) [inline x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/downsample/DownsampleActionSingleNodeTests.java:531]:
```suggestion
                fail("rollup index has not been created");
```

**Review by csoulios — APPROVED** [review]
Code looks good. I left very minor comment for a typo. Thank you

**Comment by csoulios** [comment]
@elasticmachine update branch

**Comment by weizijun** [comment]
@elasticmachine update branch

**Pull request merged**